jme3-desktop/src/main/java/jme3tools/converters/ImageToAwt.java

```diff
@@ -87,12 +88,19 @@ public DecodeParams(int bpp, int rm, int rs, int im, int is, boolean alpha){
             this.is = is;
         }
 
-        /*
-         * public DecodeParams(int bpp, int rm, int rs, int im, int is){ this(bpp, rm, rs, im, is, false); }
-         */
+        @SuppressWarnings("unused")

```

Comment:
Why is this getting a @SuppressWarnings("unused")? There might be legitimate reasons for this suppression but a comment to explain why the warnings are being supressed would be nice.

(I'm surprised a constructor has unused methods. Sometimes it makes sense for a method if it is overridden and the child class needs them)